src/pkcs1v15.rs

```diff
@@ -145,7 +143,10 @@ fn encrypt<R: CryptoRngCore + ?Sized>(
     key::check_public(pub_key)?;
 
     let em = pkcs1v15_encrypt_pad(rng, msg, pub_key.size())?;
-    let int = Zeroizing::new(BigUint::from_bytes_be(&em));
+    let int = BoxedUint::from_be_slice(
+        &em,
+        crate::traits::keys::PublicKeyParts::n_bits_precision(pub_key),
```

Comment:
Is there some reason you aren't just importing this, especially since it's used repeatedly? So it could just be...
```suggestion
        PublicKeyParts::n_bits_precision(pub_key),
```

Reply:
cleaned these up